src/Input/Input.variants.js

```diff
@@ -0,0 +1,81 @@
+import { BORDERS, RADII } from '../assets/core'
+import colors from '../assets/colors'
+import sizes from '../assets/sizes'
+import theme from '../assets/theme'
+import typography from '../Typography/variants'
+
+const DEFAULT_STYLE = {
+  ...typography.inputLabel,
+  height: sizes.inputHeight,
+  background: colors.backgroundLightGray,
+  border: `${BORDERS[1]} ${colors.inputGray}`,
+  boxSizing: 'border-box',
+  color: colors.textInputActive,
+  borderRadius: RADII[3],
+  '::placeholder': {
+    color: colors.textInputPlaceholder
+  },
+  ':hover': {
+    borderColor: colors.brandLight
+  },
+  ':focus': {
+    borderColor: colors.brandLight
+  },
+  ':disabled': {
+    background: colors.backgroundGray,
+    color: colors.textInputDisabled,
+    ':hover, :focus': {
+      borderColor: colors.inputGray
+    }
```

Comment:
On disabled you should have no pointer events. Please read [this](https://www.a11y-101.com/development/aria-disabled). It applies to buttons and inputs as well.
```suggestion
    cursor: not-allowed;
    pointer-events: none;
```